src/test/java/com/suse/saltstack/netapi/parser/JsonParserTest.java

```diff
@@ -47,4 +50,37 @@ public void testSaltStackTokenParserWrongDate() throws Exception {
         InputStream is = this.getClass().getResourceAsStream("/login_response_wrong_date.json");
         JsonParser.TOKEN.parse(is);
     }
+
+    @Test
+    public void testSaltStackJobsParser() throws Exception {
+        InputStream is = this.getClass().getResourceAsStream("/jobs_response.json");
+        Result<List<Map<String, Job>>> result = JsonParser.JOBS.parse(is);
+        assertNotNull("failed to parse", result);
+
+        Map<String, Job> jobs = result.getResult().get(0);
+        Job job = jobs.get("20150304192951636258");
+        assertNotNull(job);
+        assertEquals(Arrays.asList(), job.getArguments());
+        assertEquals("test.ping", job.getFunction());
+        assertEquals("*", job.getTarget());
+        assertEquals("glob", job.getTargetType());
+        assertEquals("chuck", job.getUser());
+
```

Comment:
Oh yeah, I've definitely problems with empty lines. (Maybe introducing checkstyle would be nice. It would save a lot of everyone's time and some reviewers' nerves).